sites/all/themes/orwell/sass/component/_facets.scss

```diff
@@ -159,6 +159,10 @@ fieldset {
   }
 }
 
+.pane-ding-facetbrowser .pane-title {
+  display: none;
```

Comment:
Giver det mening evt at fjerne det igennem panels?

Reply:
You're right.